manifests/sysconfig_init.pp

```diff
@@ -15,18 +18,31 @@
 # @param prompt
 # @param autoswap
 #
+#For all `setcolor` variables, use the following color options as a string:
+#   * default
+#   * black
+#   * red
+#   * green
+#   * yellow
+#   * blue
+#   * magenta
+#   * cyan
+#   * white
+#
+# author: SIMP Team <simp@simp-project.com>
+#
 class useradd::sysconfig_init (
-  Useradd::Bootup      $bootup            = 'color',
-  Integer              $res_col           = 60,
-  String               $move_to_col       = "\"echo -en \\\\033[\${RES_COL}G\"",
-  String               $setcolor_success  = "\"echo -en \\\\033[0;32m\"",
-  String               $setcolor_failure  = "\"echo -en \\\\033[0;31m\"",
-  String               $setcolor_warning  = "\"echo -en \\\\033[0;33m\"",
-  String               $setcolor_normal   = "\"echo -en \\\\033[0;39m\"",
-  Stdlib::AbsolutePath $single_user_login = '/sbin/sulogin',
-  Integer[1,8]         $loglvl            = 3,
-  Boolean              $prompt            = false,
-  Boolean              $autoswap          = false,
+  Useradd::Bootup         $bootup            = 'color',
+  Integer                 $res_col           = 60,
+  Optional[String]        $move_to_col       = undef,
+  String                  $setcolor_success  = 'green',
+  String                  $setcolor_failure  = 'red',
```

Comment:
I think this should be an Enum of the colors that are in the hash in the template. If a user types in an invalid color here, like 'simp', then they wouldn't know until they ran puppet and broke a system.